manual/automated_code_review.md

```diff
@@ -12,4 +12,8 @@ Codacy is free for open source, and it provides RuboCop analysis out-of-the-box.
 
 ### Hound
 
-[Hound](https://github.com/houndci/hound) is yet another alternative, but their main focus is code style.
\ No newline at end of file
+[Hound](https://github.com/houndci/hound) is yet another alternative, but their main focus is code style.
+
+### SideCI
+
+[SideCI](https://sideci.com) is yet another alternative. It improves your team's productivity by automating code analysis.
```

Comment:
I'd use some other wording here. The current text was written by someone who cares about `codacy` only - that's why the other services have pretty much no description.